Functions/xASL_io_ReadDataPar.m

```diff
@@ -1,10 +1,12 @@
-function [x] = xASL_io_ReadDataPar(pathDataPar)
+function [x] = xASL_io_ReadDataPar(pathDataPar,bDataPar)
 % xASL_io_ReadDataPar This function reads data-parameter .json or .m file, which contains data and processing settings, and creates the x structure.
 %
 % FORMAT:   [x] = xASL_io_ReadDataPar(pathDataPar)
 %
 % INPUT:
 %   pathDataPar     - Filename of the input parameter file with either .m or .json extension
+%   bDataPar        - Read JSON as dataPar.json (BOOLEAN, OPTIONAL, DEFAULT = true)
```

Comment:
I would turn it around. If DataPar is the default, then the new option is `bStudyPar`. Default is off, and special case is to turn it on.

Reply:
Sure, easy fix: 4dbefa8 👍